resources/todomvc/architecture-examples/angular/src/app/app.module.ts

```diff
@@ -1,18 +1,26 @@
 import { NgModule } from '@angular/core';
+import { ReactiveFormsModule } from '@angular/forms';
 import { BrowserModule } from '@angular/platform-browser';
-import { HttpClientModule } from '@angular/common/http';
-import { FormsModule } from '@angular/forms';
 
+import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { TodoHeaderComponent } from './todo-header/todo-header.component';
+import { TodoListComponent } from './todo-list/todo-list.component';
+import { TodoFooterComponent } from './todo-footer/todo-footer.component';
+import { TodoItemComponent } from './todo-item/todo-item.component';
 
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
+    TodoHeaderComponent,
+    TodoListComponent,
+    TodoFooterComponent,
+    TodoItemComponent
   ],
   imports: [
     BrowserModule,
-    FormsModule,
-    HttpClientModule
+    AppRoutingModule,
+    ReactiveFormsModule
   ],
```

Comment:
Note: it looks like that newer versions of Angular do not need NgModule anymore, see https://angular.io/guide/standalone-components

I wonder if we shouldn't migrate to that.

Reply:
hmm will look into it.